src/OpenApi/src/Services/Schemas/OpenApiSchemaService.cs

```diff
@@ -121,6 +122,28 @@ internal sealed class OpenApiSchemaService(
         }
     };
 
+    private static string GetBinarySchemaId(Type type)
```

Comment:
This has changed the meaning of the code. It was returning the same value for all types before, now they're all different. I doubt this is correct, but even if it is it will cause tests to fail so will need further changes.

I suggest you follow the [guide to build and test the code](https://github.com/dotnet/aspnetcore/blob/main/docs/BuildFromSource.md aspnetcore/docs/BuildFromSource.md) and change the code using an IDE and run the relevant tests so you can check none get broken, and also so you can add a test for this change.